html5/js/Client.js

```diff
@@ -2965,10 +2959,10 @@ class XpraClient {
         return;
       }
     }
-    const digest = Utilities.s(packet[3]);
```

Comment:
Have you tested with older servers without `rencodeplus`?
This call is present to ensure backwards compatibility.